packages/@orchidui-vue/src/Charts/BarChart/OcBarChart.vue

```diff
@@ -127,18 +127,31 @@ const toggleLegendName = (name) => {
   });
 };
 
+const renderChart = () => {
+  myChart.value = echarts.init(barChart.value);
+  myChart.value.setOption(options.value);
+};
+
+const resizeChart = () => {
+  myChart.value.dispose();
+  renderChart();
+};
+
 defineExpose({
   toggleLegendName,
 });
 
 onMounted(() => {
-  myChart.value = echarts.init(barChart.value);
-  myChart.value.setOption(options.value);
+  renderChart();
+
```

Comment:
hi @kutsaniuk , I'm planning to implement tailwind breakpoints into javascript. md, lg, xl, 2xl. to be the same as css. it's better to make variant props, and watch the window resize using this composable.  https://github.com/hit-pay/hitpay2020/blob/1b0a4f3e4fd1ee970a955d0d85a0343edaffcf2d/src/frontend/dashboard/resources/js/composables/useWindowWidth.js  cc @HoaiLee @mykhailo-diakovych